WISE-Community/WISE
# Upgraded karma to 4.1

Also: upgraded jasmine to 3.4, and karma-jasmine to 2.0, made site test reporter match wise5 test reporter format.

To test:
1. run npm install to download new dependencies
2. run ```npm test``` and make sure that wise5 tests pass
3. run ```npm run test-site``` and make sure that site tests pass
4. run ```npm run watch-all-wise5``` and make sure that tests run when you make a change to a *.es6 file
5. run ```npm run watch-all-site``` and make sure that tests run when you make a change to a *.ts file
